### PR TITLE
Issue #SB-30678 feat: AWS Config keys added

### PR DESCRIPTION
**Pull request by rajeshkumaravel**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-30678" title="SB-30678" target="_blank">Minor-EnhancementSB-30678</a>  CSP Implementation: Cloud Agnostic report storage implementation
  </td></table>
  <br />
 

- Issue #SB-30678 feat: AWS Config keys added

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 1]
```diff
@@ -202,3 +202,13 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
 #Release-4.6.0
 sunbird_collection_children_limit={{ sunbird_collection_children_limit | default(1200) }}
 sunbird_questionset_children_limit={{ sunbird_questionset_children_limit | default(500) }}
+
+
+#release-5.0.0
+#AWS config for storage access
+sunbird_cloud_storage_provider={{sunbird_cloud_storage_provider | default("azure")}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:209]: Use `cloud_service_provider` instead `sunbird_cloud_storage_provider`
Remove `default("azure")` value

Comment (rajeshkumaravel) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env]: Updated changes

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 2]
```diff
@@ -202,3 +202,14 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
 #Release-4.6.0
 sunbird_collection_children_limit={{ sunbird_collection_children_limit | default(1200) }}
 sunbird_questionset_children_limit={{ sunbird_questionset_children_limit | default(500) }}
+
+
+#release-5.0.0
+#AWS config for storage access
+sunbird_cloud_storage_provider={{cloud_service_provider}}
+sunbird_aws_bucket_name={{sunbird_aws_bucket_name}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:210]: Please change to `aws_private_bucket_name`

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 3]
```diff
@@ -202,3 +202,14 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
...
+sunbird_aws_reports={{sunbird_aws_reports | default("reports")}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:211]: Please change to `aws_reports_folder_name`

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 4]
```diff
@@ -202,3 +202,14 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
...
+sunbird_aws_labels={{sunbird_aws_labels | default("labels")}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:212]: Please change to `aws_labels_folder_name `

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 5]
```diff
@@ -202,3 +202,14 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
...
+sunbird_aws_access_key={{sunbird_aws_access_key | default("")}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:213]: Pleas change to `aws_private_bucket_access_key`

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 6]
```diff
@@ -202,3 +202,14 @@ sunbird_portal_uci_bot_phone_number={{sunbird_portal_uci_bot_phone_number | defa
...
+sunbird_aws_secret_key={{sunbird_aws_secret_key | default("")}}
+sunbird_aws_region={{sunbird_aws_region | default("ap-south-1")}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:215]: Pleas change to `aws_private_bucket_region`

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**